SeleniumPracticalExercise/PageObjects/EmployeeInformationPage.cs

```diff
@@ -28,36 +29,38 @@ public void AddButton()
         }
 
         /// <summary>
-        ///  txtbox Emp ID and insert stored ID
+        /// this is the second time on page where we...... 
```

Comment:
Make sure you create your XML doc *after* adding all parameters, etc... pretty much when you're one with it. That way, when you create the XML doc it has all the pieces. In this case you are missing docs for the parameter, storeEmployeeId. To add it, I usually copy out the docs that I have already, delete the rest, and then start a new doc. That should put the skeleton in for each of the parts, then paste in what you had before and fill in the rest.

Reply:
updated my XML in SetEmployeeIdField to have param in it